src/screens/CreateTokenName.js

```diff
@@ -46,11 +48,19 @@ class CreateTokenName extends React.Component {
               onChangeText={this.onNameChange}
               value={this.state.name}
```

Comment:
Should we validate that this input has at most 30 chars? Just to keep it consistent with the full node validation

Reply:
That's true, we're not enforcing it here. Will do it.

Reply:
Did that and added a 'counter' for the name:
![image](https://user-images.githubusercontent.com/3236756/64604353-e8397180-d3b9-11e9-861a-4219de8336a6.png)